### ISSUE TITLE
sort out the keymap nonsense

### ISSUE THREAD

**Issue by boneskull** [headspinio/appium-lg-webos-driver#6]
So there are keys & keycodes that the remote understands and keys & keycodes that chomedriver understands.  

Ideally, someone using the driver should be able to provide the same key/keycode regardless of how the key gets pressed.

But there isn't a 1:1 relationship, and the identifiers are not the same (whether that's because they really aren't, or because of how we've implemented).  

So:

1. Track down the list of what the remote understands
2. Cross-reference it with what chromedriver understands
3. Provide a unified, sane interface

This may cause breaking changes, but if we assume that the chromedriver strategy isn't actually being used (which may be a safe assumption), we can likely isolate the breakage to that subsystem.

I suspect that gathering the necessary information will be a combination of reading more LG dev docs and cruising GitHub for implementations.

**Comment by jlipps** [comment]
yeah this is a good idea. luckily i don't think we'd be breaking this for any serious use yet
